packages/scheduler/src/Scheduler.js

```diff
@@ -261,21 +302,45 @@ function unstable_scheduleCallback(priorityLevel, callback, options) {
       typeof options.timeout === 'number'
         ? options.timeout
         : timeoutForPriorityLevel(priorityLevel);
+    // if (enableProfiling) {
+    //   var _label = options.label;
+    //   if (typeof _label === 'string') {
+    //     label = _label;
+    //   }
+    // }
   } else {
     timeout = timeoutForPriorityLevel(priorityLevel);
     startTime = currentTime;
   }
 
   var expirationTime = startTime + timeout;
 
-  var newTask = {
-    id: taskIdCounter++,
-    callback,
-    priorityLevel,
-    startTime,
-    expirationTime,
-    sortIndex: -1,
-  };
+  var newTask;
+  if (enableProfiling) {
+    newTask = {
+      // __PROFILER__ only
+      isQueued: false,
```

Comment:
Just create all the normal fields above and then append this by mutation. Closure will know to inline it into the object literal.

Reply:
Wow I had no idea. I'll remove my change then.